### DISCUSSION TITLE
performance on the water model

### DISCUSSION THREAD

**Discussion by nisargjoshi10** [mir-group/nequip#216] [Q&A] [answered]
I'm wondering if you had the simulation performance stats w/ LAMMPS for the water model discussed in the paper. I have been trying to train the water model and run an MD simulation but the performance was pretty bad. Also if you had any suggestions for accelerating the simulation, I'd be happy to learn since for larger system size my simulation is taking an awful long time.

Thanks,
Nisarg

**Comment by simonbatzner** [comment, marked as answer]
Discussed via email.
